src/modules/mc_pos_control/mc_pos_control_params.c

```diff
@@ -668,10 +668,10 @@ PARAM_DEFINE_FLOAT(MPC_LAND_ALT2, 5.0f);
  * A time constant of 0 disables the ramp
  *
  * @min 0
- * @max 1
+ * @max 5
  * @group Multicopter Position Control
  */
-PARAM_DEFINE_FLOAT(MPC_TKO_RAMP_T, 0.4f);
+PARAM_DEFINE_FLOAT(MPC_TKO_RAMP_T, 3.0f);
```

Comment:
@MaEtUgR Do you think 3 seconds is not a bit too much?

Reply:
I found it to be a good general default value. It's more on the slower side but still nice for a racer and probably desired on any bigger vehicle. What were your impressions? These were the values we tested outside yesterday and reflected in the video from https://github.com/PX4/Firmware/pull/12014#issuecomment-494360978